client/app/app.js

```diff
@@ -23,4 +24,16 @@ angular.module('venueApp', [
       });
 
     $locationProvider.html5Mode(true);
-  });
+  })
+  .config(function(uiGmapGoogleMapApiProvider) {
+    uiGmapGoogleMapApiProvider.configure({
+        //    key: 'your api key',
```

Comment:
this needs to be in some config file- i know we don't really have a config file for the client but this should be in it- we could use angular.constant